inputfiles/patches/cssom.kdl

```diff
@@ -6,4 +6,13 @@ interface CSSStyleDeclaration \
   forward=CSSStyleDeclarationBase \
   forwardExtends=CSSStyleProperties
 
-interface CSSStyleProperties replaceReference=CSSStyleDeclaration
+interface CSSStyleProperties replaceReference=CSSStyleDeclaration {
+  property imageOrientation deprecated=#true
+  property wordWrap deprecated=#true
+  
+  // The corresponding standardized property is not supported by
+  // anyone as of 2024-10.
+  property webkitLineClamp deprecated=#false
+
+  property cssFloat mdnUrl="https://developer.mozilla.org/docs/Web/API/CSSStyleDeclaration/cssFloat"
```

Comment:
Do we lose MDN URL if we skip this one? We have https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleProperties/cssFloat so hopefully not?